src/settings_page.ts

```diff
@@ -0,0 +1,67 @@
+// Script used by the settings page.
+
+import * as Config from './config'
+import * as RC from './config_rc'
+import * as Messaging from './messaging'
+
+const rctextarea = <HTMLTextAreaElement>document.querySelector('#rc-textarea')
+const isAutoloadCheckbox =
+    <HTMLInputElement>document.querySelector('#toggle-fs-rc')
+
+// Start by showing current settings
+activateTab('#current-settings')
+fillCurrentConfig()
+fillRc()
+
+// Switch tab when a its clicked on the tabbar
```

Comment:
*Switch tab when the tab bar is clicked

or just "Switch settings tab on click"